proposals/lip-0053.md

```diff
@@ -591,7 +591,7 @@ def execute(trs: Transaction) -> None:
     # Update the context to indicate that now we stop the CCM processing.
     ctx.ccmProcessing = False
 
-    # Update sidechain validators.
```

Comment:
I would keep this comment as is because it is more precise. Even if we don't have `sidechain validator` as a term in the sheet, we do have the terms `sidechain` and `validator` separately so I think it is fine to create compound nouns to be more specific (it is just a synonym for validators of a sidechain).

Reply:
reverted everywhere